app/src/main/kotlin/com/intive/picover/images/repository/ImagesRepository.kt

```diff
@@ -0,0 +1,32 @@
+package com.intive.picover.images.repository
+
+import android.net.Uri
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+import javax.inject.Inject
+import kotlin.coroutines.resumeWithException
+import kotlinx.coroutines.suspendCancellableCoroutine
+
+class ImagesRepository @Inject constructor() {
+	private val storageRef = FirebaseStorage.getInstance().reference.child("image")
+
+	internal suspend fun fetchItems(): List<StorageReference> {
```

Comment:
Why not to use `= ...` and drop type?

Reply:
![image](https://user-images.githubusercontent.com/33199010/232217178-122f81e5-3434-4e77-bd1b-919617e319d1.png)